mmdata.c

```diff
@@ -1098,6 +1131,34 @@ nmbrString *nmbrSpace(long n)
   return sout;
 }
 
+/* Search for the nth occurrence of string2 in string1 */
+long nmbrInstrN(long start_position, long occ, nmbrString *string1,

```

Comment:
why is occ defined as long? Do you really expect more than two billion occurrences (or even 32000 should int be only 16 bit wide) of a substring in a string?